cluster/charts/crossplane/templates/crossplane-admin-clusterrole.yaml

```diff
@@ -16,6 +16,46 @@ aggregationRule:
 rules: []
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}:system:aggregate-to-crossplane-admin
+  labels:
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    crossplane.io/scope: "system"
+    rbac.crossplane.io/aggregate-to-crossplane-admin: "true"
+rules:
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - clusterrolebindings
```

Comment:
just wondering if clusterroles & roles needed as well.

Reply:
If `crossplane-admin` can modify the ClusterRoles, they can change the definition of the built-in default Crossplane roles which gives this binding rule bending powers (the rules of crossplane personas!)

The cluster-admin can grant additional permissions as needed, separate from the prescribed Crossplane persona roles.